packages/calypso-products/src/plans-list.tsx

```diff
@@ -1414,6 +1419,7 @@ const getWooExpressMediumPlanCompareFeatures = (): string[] => [
 	FEATURE_AUTOMATED_BACKUPS_SECURITY_SCAN,
 	FEATURE_AD_FREE_EXPERIENCE,
 	FEATURE_UNLIMITED_ADMINS,
+	//TODO
```

Comment:
TODO: Is there a way to view the WooExpress plans grid?

Reply:
Update: The WooExpress plans grid is not shown anywhere, so I am skipping this change for now.